express-delete/server.js

```diff
@@ -35,10 +35,11 @@ app.get('/api/grades', (req, res) => {
 app.delete('/api/grades/:id', (req, res) => {
   if (grades[req.params.id]) {
     delete grades[req.params.id];
+    res.sendStatus(204);
   } else {
     console.log(`ID of ${req.params.id} not found, please try again...`);
```

Comment:
This log message will not be seen by the client. It will only be seen by someone looking at the terminal, which in a server will probably be nobody!

If you want to send a message back to the client with the 404, use
`res.status(404).send(`ID of ${req.params.id} not found, please try again...`);`